Mathlib/CategoryTheory/WithTerminal/Basic.lean

```diff
@@ -447,6 +448,71 @@ def equivComma : (WithTerminal C ⥤ D) ≌ Comma (𝟭 (C ⥤ D)) (Functor.cons
 
 end
 
+open CategoryTheory.Limits CategoryTheory.Limits.WidePullbackShape
```

Comment:
I would suggest moving this part to a new file `WithTerminal.Discrete`, so as to reduce imports.

Reply:
I don't know. Doing so removes merely one transitive import